programs/survival/cart_breaker.sc

```diff
@@ -0,0 +1,30 @@
+// cart_breaker.sc 
+// by kergadon aka nsluhrs
+// Breaks minecarts such as chest minecarts or hopper minecarts also boats with chests
+__config()->{'stay_loaded'->true};
+
+__on_player_attacks_entity(player, entity) ->(
+  mainhand=query(player,'holds','mainhand');
+  // checks that an axe is held and player has right clicked
+  if(query(player, 'sneaking') && mainhand:0 ~ '_axe',
+    entity_name=str(entity);
+    if(entity_name ~ 'Minecart with ' || entity_name ~ 'Boat with Chest',
+      // drops all items in inventory 
```

Comment:
I don't really like checking for things using the name of an item/entity. 
I this will return a different result depending on the language settings.
So this app as is won't work when I set my language settings to Spanish for example.
ie. `Vagon con TNT`

instead you could query the `'type'` which returns the same thing regardless of language.

```
entity~'type'
tnt_minecart
```

Reply:
Suggested a change to fix this